dataland-backend/src/test/kotlin/org/dataland/datalandbackend/services/CompanyManagerTest.kt

```diff
@@ -59,52 +62,16 @@ class CompanyManagerTest(
     @Test
     fun `search for them one by one by using their names`() {
         for (company in testCompanyList) {
-            val searchResponse = testCompanyManager.searchCompaniesAndGetApiModel(
-                CompanySearchFilter(
-                    searchString = company.companyName,
-                    onlyCompanyNames = true,
-                ),
+            val searchResponse = testCompanyManager.searchCompaniesByNameOrIdentifierAndGetApiModel(
+                searchString = company.companyName,
             )
             assertTrue(
-                searchResponse.any { it.companyInformation.companyName == company.companyName },
+                searchResponse.any { it.companyName == company.companyName },
                 "The posted company could not be retrieved by searching for its name.",
             )
         }
```

Comment:
This test and also the one above have been switched from testing `searchCompaniesAndGetApiModel()` to testing `searchCompaniesByNameOrIdentifierAndGetApiModel()`.
Doesn't `searchCompaniesAndGetApiModel()` miss some coverage now?

Reply:
We rewrote those as e2e tests and the same functionabilities should be tested in there

Reply:
Ok werde ich mir anschauen, Danke für den Hinweis 👍